content/webapp/pages/events/[eventId].tsx

```diff
@@ -65,6 +65,12 @@ import { PaletteColor } from '@weco/common/views/themes/config';
 import { a11y } from '@weco/common/data/microcopy';
 import { Pageview } from '@weco/common/services/conversion/track';
 import { createPrismicLink } from '@weco/common/views/components/ApiToolbar';
+import EventsPage, {
+  Props as EventsProps,
+  getServerSideProps as gSSP,
```

Comment:
could we call this something like getServerSidePropsForPeriod, so it's more obvious why we're using it, and maybe comment in the file that the value of eventId can be a period slug rather than an actual event id

Reply:
the rewrites worked for this, so none of this ended up being necessary.
I'm glad I have this functionality, but I don't know when I'd opt for a rewrite as opposed to doing things the way in this file?

Maybe if the GSSP has universal use?